.github/workflows/publish-golang.yml

```diff
@@ -67,6 +87,10 @@ jobs:
           git config --global user.email github-actions@github.com
           git config --global user.name github-actions
           git add breez_sdk/breez_sdk.go
+          git add breez_sdk/lib/android-386/libbreez_sdk_bindings.so
+          git add breez_sdk/lib/android-aarch/libbreez_sdk_bindings.so
+          git add breez_sdk/lib/android-aarch64/libbreez_sdk_bindings.so
+          git add breez_sdk/lib/android-amd64/libbreez_sdk_bindings.so
```

Comment:
It could be that android needs `libc++_shared.so` to be available. It is also included in the downloaded archives next to `libbreez_sdk_bindings.so`.

Reply:
Good catch, seems like the any needed. I get linking errors otherwise